web/index.js

```diff
@@ -0,0 +1,189 @@
+/* eslint-disable node/no-missing-import */
+import './registerStaticFiles.js'
+import * as SvgCss from 'bundle-text:../lib/svg.css'
+import * as monaco from 'monaco-editor'
+import {config, theme, tokenizer} from './wsd_language.js'
+import Store from '../test/store.js'
+import {SyntaxError} from 'quence/lib/grammar.js'
+import {draw} from 'quence'
+
+self.MonacoEnvironment = {
+  getWorkerUrl(moduleId, label) {
+    if (label === 'json') {
+      return './json.worker.js'
+    }
+    if (label === 'css' || label === 'scss' || label === 'less') {
+      return './css.worker.js'
+    }
+    if (label === 'html' || label === 'handlebars' || label === 'razor') {
+      return './html.worker.js'
+    }
+    if (label === 'typescript' || label === 'javascript') {
+      return './ts.worker.js'
+    }
+    return './editor.worker.js'
+  },
+}
+
+const src = document.querySelector('#src')
+const dest = document.querySelector('#dest')
+const downloadSVG = document.querySelector('#downloadSVG')
+const downloadPDF = document.querySelector('#downloadPDF')
+const downloadPDFhidden = document.querySelector('#downloadPDFhidden')
+
+monaco.languages.register({id: 'wsd'})
+monaco.languages.setMonarchTokensProvider('wsd', tokenizer)
+monaco.languages.setLanguageConfiguration('wsd', config)
+monaco.editor.defineTheme('wsdTheme', theme)
+
+const editor = monaco.editor.create(src, {
```

Comment:
Put a better first example here.  Also, keep it in an external file and include it with bundle-text.